### DISCUSSION TITLE
CSVLoader and Q&A for CSV

### DISCUSSION THREAD

**Discussion by ngmisl** [langflow-ai/langflow#214] [Q&A]
What am I doing wrong here? It knows that there is a wallet with 55 of holdings, but when I ask it to find the biggest holdings it just gives plain wrong answers?

![image](https://user-images.githubusercontent.com/98217124/235159995-b8cccbf3-6820-45da-914f-2e3174e11109.png)

**Comment by ATL64** [comment]
Click on the little bubble on the top right of the message to see the agent's thoughts and actions.

**Comment by ogabrielluiz** [comment]
@ngmisl You can use a CSVAgent to make it run analytical tooling (pandas) on it.

**Comment by ATL64** [comment]
@ogabrielluiz Is there a benefit of using Chroma for CSV's? Even for lookups this doesn't look like the right tool to me.

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
